.travis.yml

```
@@ -4,7 +4,7 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
+  - 1.3
```

Will automatically update to the latest verison 1 release
```suggestion
  - 1
```

---

I think it would be helpful to add 1.0 tests for both travis and appveyor.